src/lib/ibcclient.ts

```diff
@@ -705,7 +705,7 @@ export class IbcClient {
       value: MsgUpdateClient.fromPartial({
         signer: senderAddress,
         clientId,
-        header: {
+        clientMessage: {
```

Comment:
This only works with newer versions of the SDK then?
It would be good to document compatibility

Reply:
The field was renamed in https://github.com/cosmos/ibc-go/pull/1871. But since it is the same field number in the protobuf serialization as before I assume nothing bad happens with the change 🤞

Reply:
Only difference is amino signing, then? Which likely isn't relevant here